docs/features/writing-different-telemetry-types.md

```diff
@@ -64,4 +64,23 @@ loger.LogSecurityEvent("Invalid Order", telemetryContext);
 // Output: "Events Invalid Order (Context: [EventType, Security], [OrderId, OrderId was not in correct format])"
 ```
 
+## Metrics
+
+Metrics allow you to report custom metrics which allow you to give insights on application-specific metrics.
+
+Here is how you can report an `Invoice Received` metric:
+
+```csharp
+var context = new Dictionary<string, object>
+{
+    { "InvoiceId", "ABC"},
+    { "Vendor", "Contoso"},
+};
+
+logger.LogMetric("Invoice Received", 133.37, context);
+// Output: "Metric Invoice Received: 133.37 (Context: [InvoiceId, ABC], [Vendor, Contoso])"
+```
+
+By using contextual information, you can create powerful metrics. When writing to Application Insights, for example, which will report the `Invoice Received` metric as [multi-dimensional metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-platform-metrics#multi-dimensional-metrics) which allow you to filter the metric based on its context.
```

Comment:
Ah, cool, then it's not only useful for Application Insights sinks.

Reply:
Certainly, you can use it with any sink! But will be extra powerful with Application Insights!